Makefile

```diff
@@ -59,4 +62,4 @@ build-swift-package: build
 	$(call XCODEBUILD_CMD, build)
 
 test-swift-package: build
-	$(call XCODEBUILD_CMD, test)
```

Comment:
Will this have a negative impact on CI checks? Or does the device used not matter for the tests?